src/components/Menus/WalletMenu/WalletMenu.style.ts

```diff
@@ -78,9 +78,17 @@ export const WalletCardButtonContainer = styled(Container)(({ theme }) => ({
 export const WalletCardBadge = styled(Badge)(({ theme }) => ({
   borderRadius: '50%',
   // overflow: 'hidden',
+  span: {
+    width: 44,
+    height: 44,
+    padding: 0,
+    transform: 'unset',
+    bottom: '-11.5px',
+    right: '-13.5px',
+  },
   '> .MuiAvatar-root': {
     overflow: 'hidden',
-    '--g': '#0000 98%,#000',
+    '--g': '#0000 100%,#000',
```

Comment:
I think it looks better with a bit of blur when not zoomed in. Without blur, there is no antialiasing effect 
<img width="298" alt="image" src="https://github.com/lifinance/jumper.exchange/assets/18644653/ce9d29a6-b8a8-410a-b32e-4958af2b1608">

Reply:
Also, I think we should have different masks for different image sizes. If you take a look at the small icon, there is a bigger cutout
<img width="63" alt="image" src="https://github.com/lifinance/jumper.exchange/assets/18644653/8a58b708-7ea4-4e11-be9e-de766a68af66">

Reply:
Good points! Here is some updates:

WalletCard
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/da346ba6-2944-48d5-ae67-fc79e70744b3)

WalletButton
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/740364d6-93e7-483c-927f-70452f0faf4f)

Both:
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/c89cf9a4-ad7a-437b-916a-d7d02256a3b0)

Reply:
Updated again:
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/3262a706-8247-43d7-957f-64a21af4a146)